config/export/firebase-export-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Why did this file, and the other `config/export/*` files, get added?

Reply:
This and the change to `development.yml` add a single user to the emulator up its bootup. This user as the `isAdmin` flag set to true, as without this we'd have no way to make a user an admin during testing.

The problem stems from having no way of creating an admin without first having an admin to create an admin.